### DISCUSSION TITLE
BBS hosting - or just setting manual settings

### DISCUSSION THREAD

**Discussion by Biker1bob** [bozimmerman/Zimodem#94] [Q&A]
So, I am trying to use my 3.7.1 zimodem firmware on my 8266 12F nodemcu wifi modem. 

As a dial out modem it is working great, no issues. 

To receive calls though - DO I have to turn on BBS in the +config? 
  I have tried to ATS0=1 ATS41=0 and ATA9000 for the port. I then do a AT&W to save the settings.
 Those settings get wiped out next time an ATZ happens etc.. it does not save them as per at&W

If I put the BBS mode on - then I automatically answer and start streaming.. I need the bbs software to answer the phone. If I do ATS41=0 with it, it works until a ATZ or reboot and then it is returned to 1 even if I save with AT&W

If I read the AT&W -  saving the port number is not listed as one of the things thats saved.. why not? Could we not have the port number as one of the CONFIG options?

![20230111_221748](https://user-images.githubusercontent.com/33105915/211968019-23dd0bac-32f2-469f-9da5-e83e468d8af1.jpg)

**Comment by Technoid6502** [comment]
I've been playing with a couple of Zimodem I built for fun.  The ATS44= command is indispensible!  It throttles the effective input baudrate with pretty fine granularity.  ATS44=2 for example, allows my board's upload routine enough breathing room to function at 9600bps.  

As far as using it for a BBS, there's a bug in the thing which causes ARP problems.  It dials out great, but isn't reliable as a listener.  If I flush my ARP cache on my PC and then add the device manually I can ping it for awhile, but the ARP poison flows quickly and floods you out.

So, file transfers work great in RAW mode if you set the ATS44 variable appropriately.

best,
jeff

**Comment by Technoid6502** [comment]
Oh, I forgot to answer your question.  I overlooked the BBS part in AT+CONFIG and relied on the BBS's AT setup string to set up the port for listening:  ATA10001 in my case.  Unfortunately the device is broken as a listener, at least on my network.

**Reply by Technoid6502** [reply]
Both of my Zimodem were built with 8266.  It might be that the 'Guru'  build running on esp32 may not have the ARP bug.  I plan to build one in order to test the theory.

**Comment by Biker1bob** [comment]
I see the readme is now up to 4.0 I do not see a new version of the firmware though.  Has the BBS listener part of the firmware been fixed? ANy news?
